src/components/Main/Footer.tsx

```diff
@@ -1,25 +1,18 @@
 import React from 'react'
 import { Container, Image, Segment } from 'semantic-ui-react'
+import strings from '../../utils/constants'
 
 const Footer: React.FC = () => (
-  <Container style={inlineStyles.container}>
-    <Segment style={inlineStyles.segment}>
-      <Image centered size="mini" src="/images/logo-32x32.png" />
-    </Segment>
+  <Container id="footer">
+    <div id="footer-content">
+      <Image src="/images/logo-32x32.png" />
```

Comment:
Question: when this is built do we need to change where to get this images from?

Reply:
Not sure.

Reply:
I'll add an issue for that.